sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -319,6 +318,45 @@ class CudfMin(ref: Expression) extends CudfAggregate(ref) {
   override def toString(): String = "CudfMin"
 }
 
+class CudfTDigest(
```

Comment:
If this is not going to be used anywhere except in GpuApproximatePercentile it might be good to move it there.

Reply:
Done